webaverse/app
# Naruto run fix

This PR is solving 2 issues:
https://github.com/webaverse/app/issues/1938
https://github.com/webaverse/app/issues/1918

Steps for testing:
Case 1
Press C for crouch
Activate Naruto run with Shift and then double press on button W


* Naruto run should not be possible from crouch mode

Case 2:
Press and hold shift key
double press W or A or S or D and Naruto run should be activated

Double press on Shift key should not activate Naruto run
